test/loader-test.ts

```diff
@@ -118,6 +119,8 @@ describe("LoaderResolver.find(path)", () => {
 
 describe("LoaderResolver.getSourceFileHash(path)", () => {
   const time = new Date(Date.UTC(2023, 11, 1));
+  before(() => setCurrentDate(null));
+  after(() => setCurrentDate(new Date("2024-01-10T16:00:00")));
```

Comment:
This seems highly undesirable since this test now has the side-effect of setting the current date _for all tests that run subsequently_. I’m investigating why this is needed.